containers/Dockerfile.microcosm

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.7
+MAINTAINER <>
+
+RUN apt-get -y update
+RUN apt-get -y install curl iproute2 netbase vim screen openssl
+
+ARG WEB_DOMAIN=dev.microco.sm
```

Comment:
👍 gotcha. Feel free to tweak / merge and I'll catchup later - I'm in Colorado timezone + working.